lib/src/dom/async/wait_for.dart

```diff
@@ -168,7 +168,7 @@ Future<T> waitFor<T>(
 Future<void> waitForElementToBeRemoved(
   Node Function() callback, {
   Node container,
-  Duration timeout,
+  Duration timeout = const Duration(seconds: 1),
```

Comment:
Is this something we can add test coverage for to ensure that it times out gracefully?

Reply:
Yup, good call. On it